src/components/Navbar.tsx

```diff
@@ -105,7 +105,7 @@ function Navbar({ scrollToExplore }: { scrollToExplore: any }) {
             preview={false}
             style={{
               paddingRight: screens.md ? "1.5em" : "10px",
```

Comment:
okay @nitro56565 can you change this: 1.5em in the paddingRight, to the equivalent of 24 pixels from the base format  16px.

so change 1.5em to 24px to ensure consistency here!

Reply:
alright I'll do that

Reply:
@Vinyl-Davyl done with these changes